content/applications/hr/payroll/contracts.rst

```diff
@@ -118,23 +126,24 @@ A pop-up window appears with all the contract details. Modify the fields for the
    :align: center
    :alt: Edit the details for the contract.
 
-- :guilabel:`Tags`: Select any tags associated with the contract.
-- :guilabel:`Signed Document Workspace`: This is where the signatures are stored. Choose a
+- :guilabel:`Tags`: select any tags associated with the contract.
+- :guilabel:`Signed Document Workspace`: this is where the signatures are stored. Choose a
   pre-configured workspace or create a new one.
-- :guilabel:`Signed Document Tags`: Select or create any tags associated only with the signed
+- :guilabel:`Signed Document Tags`: select or create any tags associated only with the signed
   contract as opposed to the original unsigned contract.
-- :guilabel:`Redirect Link`: Enter a redirect link for the employee to access the contract. A
+- :guilabel:`Redirect Link`: enter a redirect link for the employee to access the contract. A
   redirect link takes the user from one URL to another, in this case, to the newly updated contract
   specifically written for them.
-- :guilabel:`Who can Sign`: Select either :guilabel:`All Users` or :guilabel:`On Invitation`.
+- :guilabel:`Who can Sign`: select either :guilabel:`All Users` or :guilabel:`On Invitation`.
 
-  - :guilabel:`All Users`: Any user in the organization can sign the contract.
-  - :guilabel:`On Invitation`: Only users selected in this field can sign the contract.
+  - :guilabel:`All Users`: any user in the organization can sign the contract.
+  - :guilabel:`On Invitation`: only users selected in this field can sign the contract.
 
-- :guilabel:`Invited Users`: Select the person(s) that can sign the document.
-- :guilabel:`Document`: The attached document can be replaced by clicking the :guilabel:`✏️
-  (pencil)` icon. A pop-up window appears so another document can be selected for upload. The file
-  must be a PDF. To remove the document, click the :guilabel:`🗑️ (trash can)` icon.
+- :guilabel:`Invited Users`: select the person (or people) that can sign the document.
+- :guilabel:`Document`: the attached document can be replaced by clicking the :icon:`fa-pencil`
+  :guilabel:`(pencil)` icon. A pop-up window appears so another document can be selected for upload.
+  The file must be a PDF. To remove the document, click the :icon:`fa-trash-o` :guilabel:`(trash
+  can)` icon.
 
 Once the edits are complete, click the :guilabel:`Save` button. All the information for the selected
 contract template populates the fields in the :guilabel:`Salary Information` tab. Any additional
```

Comment:
I'm not seeing any of the mentioned fields in the 'Salary Information' tab on the 15.0 runbot I am using for the review...
![Screenshot 2024-05-15 112704](https://github.com/odoo/documentation/assets/73958186/c847eb2d-3fd3-44cf-8284-2d28b7a97db2)